compiler/src/main/scala/org/scalajs/core/compiler/GenJSExports.scala

```diff
@@ -188,6 +195,44 @@ trait GenJSExports extends SubComponent { self: GenJSCode =>
             s"Exported $kind $jsName conflicts with ${alts.head.fullName}")
       }
 
+      genMemberExportOrDispatcher(classSym, jsName, isProp, alts,
+          isDispatcher = false)
+    }
+
+    private def genJSClassDispatcher(classSym: Symbol, name: String): js.Tree = {
+      var alts: List[Symbol] = Nil
+      for {
+        sym <- classSym.info.members
+        if !sym.isBridge && jsNameOf(sym) == name
+      } {
+        val tpe = sym.tpe
+        if (!alts.exists(alt => tpe.matches(alt.tpe)))
+          alts ::= sym
+      }
```

Comment:
Doesn't look like we have that anywhere else:
https://github.com/scala-js/scala-js/search?utf8=%E2%9C%93&q=isBridge
https://github.com/scala-js/scala-js/search?utf8=%E2%9C%93&q=matches